code/datums/shuttles.dm

```diff
@@ -627,6 +634,20 @@
 	admin_notes = "Has an on-board experimental cloner that creates copies of its user, alien surgery tools, and a void core that provides unlimited power."
 	credit_cost = 8000
 
+/datum/map_template/shuttle/emergency/casino
+	suffix = "casino"
+	name = "The Emergency Casino"
+	description = "Makes you feel like Space Vegas. Comes fully equiped."
+	admin_notes = "It's a normal shuttle but full of slot machines and made out of gold."
+	credit_cost = 5000
+
+/datum/map_template/shuttle/emergency/ragecage
+	suffix = "ragecage"
+	name = "THE RAGE CAGE"
+	description = "WE ARE RAGING! WE ARE CAGING! TIME TO UNLEASH ALL THE STRESS GENERATED DURING THE SHIFT! Comes with a Brig, Medbay and Cockpit included."
```

Comment:
Tone down the text a little bit. This is a roleplay server and feature still need to make sense in the world.